src/bp/ui-shared/src/Toaster/index.tsx

```diff
@@ -49,12 +50,14 @@ const warningOptions = {
   key: 'warning'
 }
 
-const dismiss = (key: string) => {
+const selectToaster = (options?: IToasterProps) => (options ? Toaster.create(options) : defaultToaster)
```

Comment:
Our custom style is now gone. Please make sure you add style.toaster by default in option object.